span.go

```diff
@@ -578,6 +582,8 @@ type SpanData struct {
 	// Context describes the context in which span occurs.
 	Context SpanContext
 
+	Links []SpanLink
```

Comment:
I think we should we unexport this. The spec says:

> The public API MUST NOT allow adding span links *after* span creation.

Reply:
Good catch! Fixed!